### DISCUSSION TITLE
Is it possible to limit the number of incoming connection to just 1?

### DISCUSSION THREAD

**Discussion by VadimIlyasov** [newcat/baklavajs#254] [Q&A]
Hi,

Is it possible to limit the number of connections that can be made to a node? I would like like to limit it to improve "readability", I think it should be fine to have multiple connections for outputs.
Thank you!

**Comment by newcat** [comment]
Yes, this is possible by using preventable events: https://v2.baklava.tech/event-system.html#events

Whenever the user wants to create a new connection, this event is called and you can check, whether the target node already has the maximum number of connections and prevent the creation of the connection in this case.
